pkg/server/routes.go

```diff
@@ -26,6 +26,7 @@ func (srv *RegistrationServer) SetupRoutes() error {
 		authConfigCtrl := controller.NewAuthConfig()
 		analyticsCtrl := controller.NewAnalytics()
 		signupCtrl := controller.NewSignup(srv.application)
+		//metricsCtrl := controller.NewMetrics()
```

Comment:
is it still needed?

Reply:
removed in [631fdc6](https://github.com/codeready-toolchain/registration-service/pull/342/commits/631fdc67dfc3b7aadcf3233ef1ecff22d2455b14)